### DISCUSSION TITLE
Lomse log file

### DISCUSSION THREAD

**Discussion by denisgottardello** [lenmus/lomse#395] [General]
Is it possible to disable log file without to edit the source code?

**Comment by cecilios** [comment]
To change log destination or to suppress it you dont have to recompile Lomse library. It is enough to change your application code. Nevertheless I suposse you can always redirect your application log stream to stderr or to a null stream whithout having to recompile your code.

See this documentation:   https://lenmus.github.io/lomse/page-logging.html
